tests/cypress/support/commands.js

```diff
@@ -15,7 +15,7 @@ Cypress.Commands.add('login', (username = Cypress.env('user'), password = Cypres
     cy.get('[placeholder="Username"]').type(username);
     cy.get('[placeholder="Password"]').type(password);
     cy.get('[type="submit"]').click();
-    cy.url().should('include', '/tasks');
+    cy.url().should('include', '/tasks').and('not.include', '/auth/login');
```

Comment:
Have you tried matching by regex instead? Something like: should('match', /\/tasks$/)

Reply:
Search replaced on regex.